src/redux/actions/samples/deleteSamples.js

```diff
@@ -33,6 +37,20 @@ const sendDeleteSamplesRequest = async (projectUuid, experimentId, sampleUuids)
   );
 };
 
+const sendDeleteSamplesRequestApiV2 = async (experimentId, sampleUuids) => {
```

Comment:
I see that sometimes the code naming assumes that no version means v1 (for example, in `sendDeleteSamplesRequest` function name) and sometimes, no version name means v2 (for example, the `SAMPLES_DELETE` and `SAMPLES_DELETE_API_V1` action names). Can you be consistent with this? This difference creates bigger scope for errors and more complexity (every time the code reader has to check both versions of a function/variable and its implementations to see what version it is, unless is has v1 or v2 appended to its name). I suggest for both v1- and v2- specific things, append v1 and v2 to the name respectively